app/containers/Header/Profile.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as routes from 'routes-config';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import Dropdown from 'components/Dropdown';
+import Li from 'components/Li';
+import Ul from 'components/Ul';
+import AddText from 'components/AddText';
+import A from 'components/A';
+
+const Img = styled.img`
```

Comment:
Not going to reuse that component anywhere else?

Reply:
not yet